explain/setup/heroku/_01_public.mdx

```diff
@@ -0,0 +1,29 @@
+# Check if auto_explain already enabled
+
+First, to check if `auto_explain` is already enabled on your system, run the
+following query with `heroku pg:psql`:
```

Comment:
With Crunchy, we say "run the following query as a Postgres superuser:", but I thought it's not obvious, so I changed it to `heroku pg:psql`